src/components/Header/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Logo from '../Logo';
+import headerBackground from '../../assets/images/header-bg.png';
+
+const HeaderTop = styled.div`
```

Comment:
Usually, we keep all styled components for each component in a separate file, so you could move all styled from this file to styled.header.jsx for example, and the same for all other files